aion/lib/aion/ranking.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Aion.Ranking do
+  @moduledoc """
+  This module provides logic for generating rankings
+  """
+  import Ecto.Query, only: [from: 2]
+  alias Aion.{Repo, User, UserCategoryScore}
```

Comment:
Consider doing it as a multiline import

Reply:
do you want to set up aliases line by line?

Reply:
No, I want to do it as it is currently done in `RoomChannel`:
```

  alias Aion.{
    Presence,
    RoomChannel.Monitor,
    QuestionChronicle,
    UserSocket
  }

```

Reply:
In my opinion, considering our current status and deadlines, this is just a cosmetic change and we probably should be more focused right now on delivering good code (not perfect one) that brings some business value ;) Things like that often stops deploying stuff for a day or two. Just my point of view :P

Reply:
Yeah I agree, it is only asking about hitting enter key three times and I don't think this is actually the thing that is slowing down this PR that much. I think that if we introduce some convention we should keep that and PRs are good places to catch these things.
 
As for the architectural changes etc. I keep that in mind and I don't ask for things that would require too much work considering the approaching deadlines :)